src/Internal/UriValidation/UriValidation.cpp

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "UriValidation.h"
+
+namespace AppInstaller::UriValidation
+{
+    UriValidationResult UriValidation(const std::string&)
+    {
+        // In Dev mode, allow all URIs
+        return UriValidationResult(UriValidationDecision::Allow, std::string());

```

Comment:
Needs test consideration, like have specific URIs that trigger specific results.